src/models/player.js

```diff
@@ -20,6 +24,17 @@ export const createPlayer = (user, options = {}) => ({
   user,
 });
 
+export const createPlayerEntities = (boardSize, users) =>
+  compose(
```

Comment:
i've seen you have used pipe somewhere else. i think you should be consistent and choose one method